src/utils/searchUtils/searchUtils.ts

```diff
@@ -133,17 +134,19 @@ export function getAntsRequiringUpdate({
   ants,
   userAddress,
 }: {
-  ants: Record<
-    string,
-    { state: AoANTState | null; handlers: AoANTHandler[] | null }
-  >;
+  ants: Record<string, ANTProcessData>;
   userAddress: string;
 }): string[] {
   return Object.entries(ants)
     .map(([id, ant]) => {
       // if user is not the owner, skip
       if (!ant.state?.Owner || ant?.state.Owner !== userAddress) return;
-      if (!AntHandlerNames.every((handler) => ant.handlers?.includes(handler)))
+      if (
+        !ant.processMeta ||
+        ant.processMeta.tags.find(
+          (t) => t.name === 'Module' && t.value !== ANT_MODULE_ID,
```

Comment:
Needs to be updated for semantic versioning

Reply:
updated this to provide a module id that is set to our state